### DISCUSSION TITLE
How to make theme.js less heavy?

### DISCUSSION THREAD

**Discussion by RoneoOrg** [Lednerb/bilberry-hugo-theme#428] [General]
Hi there,

I've been surprised by the file `/static/theme.js`: it seems to be minified but still, **1.7Mo**!

I couldn't understand precisely the content, even with an automatic reformatting in Vscode.

Is this file documented somewhere? (I searched this whole repo but found nothing)

Is there a way to disable some parts of this JS, even if some features are affected?

**Comment by RoneoOrg** [comment]
(I'm aware of the param `enableMomentJs`)

**Reply by igor-baiborodine** [reply]
Here is the readable `theme.js` file: https://github.com/Lednerb/bilberry-hugo-theme/blob/master/assets/js/theme.js, and it's self-explaining.
